tests/test_ac_harness_shared_memory.py

```
@@ -0,0 +1,205 @@
+    assert snap.is_live is True
+
+
+def test_parse_graphics_decodes_in_pit_true():
+    snap = parse_graphics(_graphics_bytes(packet_id=1, status=3, is_in_pit=True))
+    assert snap.status is AcGameStatus.PAUSE
+    assert snap.is_in_pit is True
+    assert snap.is_live is False
+
+
+@pytest.mark.parametrize(
+    "raw,expected",
```

Ruff (0.15.15)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)